client/post-editor/media-modal/index.scss

```diff
@@ -28,6 +28,7 @@
 .editor-media-modal .dialog__content {
 	position: static;
 	color: $gray-dark;
+	height: 100%;
```

Comment:
I didn't find any regressions in editor media modal. This really helps with image editor styling inside and outside editor media modal (see http://calypso.localhost:3000/devdocs/blocks/image-editor)